source/tutorials/longitudinal.rst

```diff
@@ -114,5 +114,61 @@ Second, control charts display the mean value of "metric" at each "state". The f
 
 This visualizer currently only generates control charts, which are a useful **qualitative** approach for visually identifying abnormal time points; accompanying statistical tests may be added in future releases.
 
+
+Non-parametric microbial interdependence test (NMIT)
+----------------------------------------------------
+Within microbial communities, microbial populations do not exist in isolation but instead form complex ecological interaction webs. Whether these interdependence networks display the same temporal characteristics within subjects from the same group may indicate divergent temporal trajectories. NMIT evaluates how interdependencies of features (e.g., microbial taxa, sequence variants, or OTUs) within a community might differ over time between sample groups. NMIT performs a nonparametric microbial interdependence test to determine longitudinal sample similarity as a function of temporal microbial composition. For each subject, NMIT computes pairwise correlations between each pair of features. Between-subject distances are then computed based on a distance norm between each subject's microbial interdependence correlation matrix. For more details and citation, please see `Zhang et al., 2015`_.
+
+.. note:: NMIT, as with most longitudinal methods, largely depends on the quality of the input data. This method will only work for longitudinal data (i.e., the same subjects are sampled repeatedly over time). To make the method robust, we suggest a minimum of 5-6 samples (time points) per subject, but the more the merrier. NMIT does not require that samples are collected at identical time points (and hence is robust to missing samples) but this may impact data quality if highly undersampled subjects are included, or if subjects' sampling times do not overlap in biologically meaningful ways. It is up to the users to ensure that their data are high quality and the methods are used in a biologically relevant fashion.
+
+.. note:: NMIT can take a long time to run; the following command takes around an hour to run. Just download the results file displayed below and skip ahead if you want to see how NMIT data are used downstream in QIIME2.
```

Comment:
We should show the example in genus level and screen out OTUs who have 0s in most of counts. 

In the paper, we only considered OTUs who have relative abundance >0.1% in more than 15% of the total samples. That result in 23 major taxa out of 498 taxa in genus level.

Reply:
Thanks @elong0527 ! Do you want to send me the files that you used in the paper? We could just upload these very same files into this tutorial.

Reply:
I think we can use the current tutorial data instead of maintaining two sets of data. 

That will be great if you could add one more step to make the taxa table in Genus level. If it is slow to run (true in R), we can just use the current Species level (I guess) taxa table to illustrate NMIT. 

The screening step has been implemented in the code now. 

[nbokulich/q2-intervention/pull/3#issuecomment-331730846](https://github.com/nbokulich/q2-intervention/pull/3#issuecomment-331730846)

Reply:
Hi @elong0527 the tutorial data that we use currently (species-level data) are only used for the NMIT example. So I do not have any problem with replacing this file with a genus-level taxa feature table (including the additional filtering that you describe). Thanks!